src/connection.rs

```diff
@@ -34,6 +37,8 @@ impl std::fmt::Debug for Connection {
             ConnectionImpl::Tcp(_) => f.write_str("Connection(tcp)"),
             #[cfg(all(feature = "unix", unix))]
             ConnectionImpl::Unix(_) => f.write_str("Connection(unix)"),
+            #[cfg(all(feature = "vsock", target_os = "linux"))]
```

Comment:
Shall `android` be also added?

Custom kernels may enable `AF_VSOCK`.

Reply:
> #[cfg(any(linux_android, apple_targets))] used in nix for AF_VSOCK definition

That means something similar should be available in `tokio-listener`.

If `vsock` and `tokio-vsock` supports a platform, `tokio-listener` should as well.

Reply:
nix have following aliases, https://github.com/nix-rust/nix/blob/master/build.rs#L21
And vsock exported on `any(android_linux, apple_targets)`.

Should I bring similar aliases, or just use `any(target_os = "android", target_os="linux", target_os="macos")`?
(that set which I have in not-pushed-yet development state at the moment)

Reply:
Bringing in `cfg_aliases!` (and `build.rs`) is probably too much for this. Let's wait when such feature appears in Rust proper.

The simple (latter) one looks better.